magento/magento2
# fixed issue #23024 : Grid Action Column Design Issues

### Description (*)

If grid action column position change then multi action box not align with action.

### Fixed Issues (if relevant)

1. #23024 : Grid Action Column Design Issues

### Manual testing scenarios (*)

1 . Open any Grid that action column has more than one action like CMS Block grid.
2. Programmatically change the position of an action column.
3. Comment updated_time column flush cache and reload grid.
4. Now remove comment and flush cache and reload the grid. now action column will at second last position. https://prnt.sc/ntyan7
5. Now select action, you will see the column popup always at last position. But after updating it will display with action column.

https://prnt.sc/ntybge

### Questions or comments


### Contribution checklist (*)
